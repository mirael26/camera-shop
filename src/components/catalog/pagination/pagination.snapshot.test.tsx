import { renderWithReduxAndRouter } from '../../../test/helpers/render-with-redux-and-router';
import Pagination from './pagination';

jest.mock('react-router-dom', () => {
  return {
    ...jest.requireActual('react-router-dom'),
    useParams: () => ({ page: '1' }),
  }
});

test('Pagination renders correctly', () => {
  const { asFragment } = renderWithReduxAndRouter(<Pagination pageCount={3}/>);
    expect(asFragment()).toMatchSnapshot();
});
