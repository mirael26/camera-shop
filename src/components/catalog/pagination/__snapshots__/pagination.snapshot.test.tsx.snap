// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pagination renders correctly 1`] = `
<DocumentFragment>
  <div
    class="pagination"
    data-testid="pagination"
  />
</DocumentFragment>
`;
